site/ipam.md

```diff
@@ -22,7 +22,35 @@ You can see which address was allocated with
     host1# weave ps $C
     a7aff7249393 7a:51:d1:09:21:78 10.2.3.1/24
 
-The `-iprange` parameter is given in [CIDR
+You may wish to run containers on different subnets, as described
+[here](features.html#application-isolation). To set a default subnet,
```

Comment:
make "run containers on different subnets" the link and ditch the "as described here".